test/ad/flows.jl

```diff
@@ -1,26 +1,30 @@
 @testset "PlanarLayer" begin
+    # TODO(mhauru) Remove the EnzymeReverseCrash marks once this has been fixed:
+    # https://github.com/EnzymeAD/Enzyme.jl/issues/2029
```

Comment:
these crashes and related should all be resolved

Reply:
Marked as unbroken. Likewise marked a few others.